extension/src/debug.ts

```diff
@@ -155,7 +158,7 @@ export class ASDebugSession extends LoggingDebugSession
 		// make sure to 'Stop' the buffered logging if 'trace' is not set
 		logger.setup(args.trace ? Logger.LogLevel.Verbose : Logger.LogLevel.Stop, false);
 
-		unreal.connect();
+		unreal.connect(args.port !== undefined ? args.port : this.port);
```

Comment:
not super happy with how this turned out but couldnt come up with a nicer way non-invasive way